pydicom/valuerep.py

```diff
@@ -894,6 +894,329 @@ def __bool__(self) -> bool:
 
         return bool(self.original_string)
 
+    @staticmethod
+    def _encode_component_groups(
+        alphabetic_group: SequenceType[Union[str, bytes]],
+        ideographic_group: SequenceType[Union[str, bytes]],
+        phonetic_group: SequenceType[Union[str, bytes]],
+        encodings: Optional[List[str]] = None,
+    ) -> bytes:
+        """Creates a byte string for a person name from lists of parts.
+
+        Each of the three component groups (alphabetic, ideographic, phonetic)
+        are supplied as a list of components.
+
+        Parameters
+        ----------
+        alphabetic_group: SequenceType[Union[str, bytes]]
+            List of components for the alphabetic group.
+        ideographic_group: SequenceType[Union[str, bytes]]
+            List of components for the ideographic group.
+        phonetic_group: SequenceType[Union[str, bytes]]
+            List of components for the phonetic group.
+        encodings: Optional[List[str]]
+            A list of encodings used for the other input parameters.
+
+        Returns
+        -------
+        bytes:
+            Bytes string representation of the person name.
+
+        Raises
+        ------
+        ValueError:
+            If any of the input strings contain disallowed characters:
+            '\\' (single backslash), '^', '='.
+        """
+        from pydicom.charset import encode_string, decode_string
+
+        def enc(s: str) -> bytes:
+            return encode_string(s, encodings or [default_encoding])
+
+        def dec(s: bytes) -> str:
+            return decode_string(s, encodings or [default_encoding], [])
```

Comment:
Can you change this to `decode_bytes` please?

Reply:
addressed with 7324a6a